eng/AfterSigning.targets

```diff
@@ -0,0 +1,22 @@
+<Project>
+
+  <PropertyGroup>
+    <_SuppressSdkImports>false</_SuppressSdkImports>
+  </PropertyGroup>
+
+  <Target Name="PopulateGenerateChecksumItems"
+          AfterTargets="Build"
+          BeforeTargets="GenerateChecksums" >
+
+    <ItemGroup>
+      <NuGetPackageFile Include="$(ArtifactsPackagesDir)**\*.nupkg" />
```

Comment:
Will this generate a checksum file for every nuget package? Is that needed?

Reply:
Yes, it will generate a checksum file for each NuGet package. Scoping it to just dotnet-monitor is possible (pattern match on dotnet-monitor), but given that the checksum generation is fast and there are no more than 20 NuGet packages, I didn’t think it was necessary.

Reply:
I can filter to just dotnet-monitor so there is less processing and storage if you’d like. The only minor thing that I can of as a downside is if someone is trying to figure out why only one checksum file is getting generated for one NuGet package and not others, they might not easily figure that out.